### DISCUSSION TITLE
[@types/nightwatch] Missing support for accessible globals and external globals

### DISCUSSION THREAD

**Discussion by reallymello** [DefinitelyTyped/DefinitelyTyped#59412] [Issues with a @types package] [answered]
In Nightwatch.js you can add name: value pairs in the globals collection through either the globals_path in config or through the globals property inside the test settings like this example below

`test_settings: {
default: {
globals: {
username: "myuser",
password: "password 
}
}
}`

and then access those values using syntax like `browser.globals.username` in the tests. However, when using the nightwatch types package in a typescript version of the test there is an error "Property 'username' does not exist on type 'NightwatchGlobals'"

[a comment by an automated account is not shown]

**Comment by vaibhavsingh97** [comment, marked as answer]
Thanks for raising the concern. Fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59417
